doc/15-troubleshooting.md

```diff
@@ -1,5 +1,51 @@
 # Icinga 2 Troubleshooting <a id="troubleshooting"></a>
 
+## Icinga starts/restarts/reloads very slowly
+
+### If you run Linux and systemd
+
+On RHEL or derivates add the EPEL repository first.
```

Comment:
Does Fedora count as a derivative? All other have this as part of their installation instructions already (and I think installing icinga2 would fail with missing dependencies if you haven't added it).

Reply:
Rocky 9 has Boost 1.75 in _appstream_. I.e. no, it's not required.